doc/docs/overview.md

```diff
@@ -0,0 +1,36 @@
+# Overview and user interface
+
+After starting Board Configurator, the main application window is displayed.
+
+![Board Configurator application window](./screenshots/board_ctrl_overview_alt.png "Board Configurator application window")
+
+The available options and information change after you **Select Device**.
+
+## Select Device
+
+Opens the drop-down with the list of devices connected to the computer. You can choose a device from the list of connected devices to perform further actions on the device such as programming.
+
+## Actions
+
+When you select a device, the following actions become available in the side panel:
+
+- **Write config** - Writes the selected configurations to the development kit.
```

Comment:
@cybic , is this the only action we get to use in this version? If so, are you going to hide the placeholders for other actions?

Reply:
The placeholders have already been hidden, so yes!